apps/frontend/src/app/5_pages/MarketMakingPage/components/AmbientMarketMaking/hooks/useGetAmbientPositions.ts

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
+
+import { BOB_CHAIN_ID } from '../../../../../../config/chains';
+
+import { BOB_INDEXER } from '../../../../../../constants/infrastructure';
+import { useFetch } from '../../../../../../hooks/useFetch';
+import { AmbientPosition } from '../AmbientMarketMaking.types';
+
+const account = '0xDFEC0328a07C399A7e32364DEfd3bE6aaB9365D3';
+const base = '0x0000000000000000000000000000000000000000';
+const quote = '0x6b175474e89094c44da98b954eedeac495271d0f';
+const poolIdx = '420';
+const chainId = '0x1';
+
+export const useGetAmbientPositions = () => {
```

Comment:
Pass instance of `AmbientLiquidityPool` as hook argument and retrieve needed data there.
Maybe having poolIdx in the AmbientLiquidityPool would be good idea too (for bob chain it's 3600).